### ISSUE TITLE
Permutation-based identification of important biomarkers for complex diseases via machine learning models

### ISSUE THREAD

**Issue by cmclean5** [cmclean5/PublicHealthModels#7]
## TL;DR
This paper propose a Permutation-based Feature Importance Test (PermFIT) for estimating and testing the feature importance, and for assisting interpretation of individual feature in complex frameworks, including deep neural networks, random forests, and support vector machines.

### Paper Link

https://www.nature.com/articles/s41467-021-22756-2

### Author/Institution

Xinlei Mi (Northwestern University), Baiming Zou (University of North Carolina), Fei Zou (University of North Carolina), Jianhua Hu (Columbia University)

## Overview

This paper propose a Permutation-based Feature Importance Test (PermFIT) for estimating and testing the feature importance, and for assisting interpretation of individual feature in complex frameworks, including deep neural networks, random forests, and support vector machines. PermFIT is applied to the Cancer Genome Atlas kidney tumor data and the HITChip atlas data, and demonstrate usage in identifying important biomarkers and boosting model prediction performance.

## Cite

Mi, X., Zou, B., Zou, F. et al. Permutation-based identification of important biomarkers for complex diseases via machine learning models. Nat Commun 12, 3008 (2021). https://doi.org/10.1038/s41467-021-22756-2

## Comments

Python code available: PermFIT (available at https://github.com/SkadiEye/deepTL)
